src/app/zap-templates/zcl/data-model/chip/drlc-cluster.xml

```diff
@@ -182,7 +182,7 @@ limitations under the License.
     <item fieldId="4" name="RandomDurationMinutes" type="int8u" isNullable="true" min="0x00" max="0x3C"/>
   </struct>
 
-  <cluster>
+  <cluster apiMaturity="provisional">
```

Comment:
for some reason this does not show up in the matter file. Could you check if some regen or typo issue and if not assign a bug to me?

Reply:
I think I forgot to run regen after adding that... Doing that now.

Reply:
Yep, that fixed it.